RecoLocalTracker/SiStripClusterizer/plugins/alpaka/SiStripClustersToLegacyAlgo.dev.cc

```diff
@@ -0,0 +1,187 @@
+
+#include <alpaka/alpaka.hpp>
+
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/workdivision.h"
+
+#include "SiStripClustersToLegacyAlgo.h"
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE::sistrip {
+  using namespace ::sistrip;
+  using namespace sistripConverter;
+  using namespace cms::alpakatools;
+
+  class SiStripConvKer_fillClCollSlim {
+  public:
+    template <typename TAcc, typename = std::enable_if_t<alpaka::isAccelerator<TAcc>>>
+    ALPAKA_FN_HOST_ACC void operator()(TAcc const& acc,
+                                       const uint32_t nbClusterCandidates,
+                                       SiStripClusterConstView fullCollection,
+                                       SiStripClustersSlimView slimCollection) const {
+      //
+      // for (uint32_t i=0; i<nbClusterCandidates; ++i) {
```

Comment:
I this these commented-out lines can be removed.

While you may keep the ones below if you think they may be useful for some quick debug in the future.